crox/src/main.rs

```diff
@@ -180,7 +180,8 @@ fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
             .find(" --crate-name ")
             .map(|index| index + 14);
         if let Some(index) = index_of_crate_name {
-            let (_, last) = data.metadata().cmd.split_at(index);
+            let metadata = data.metadata();
+            let (_, last) = metadata.cmd.split_at(index);
```

Comment:
Is this still needed after the OnceCell change?

Reply:
Nope! Changed back.